intelmq/bots/collectors/microsoft/collector_azure.py

```diff
@@ -23,7 +23,7 @@
     create_configuration = None  # noqa
 
 
-class MicrosoftAzureCollectorBot(CollectorBot, CacheMixin):
+class MicrosoftAzureCollectorBot(CollectorBot, CacheMixin, HttpMixin):
```

Comment:
Is it used here? The bot uses the azure library and the parameters `http_proxy` and `https_proxy` are used by direct access.